test_regress/t/t_randomize_method_with.pl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2024 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(simulator => 1);
+
+compile(
+    # Ensure we test captures of static variables
+    verilator_flags2 => ["--fno-inline"],
+    );
+
+execute(
+    check_finished => 1,
+    );
+
+ok(1);
+1;
```

Comment:
@wsnyder can we try to enforce that test needing z3 are skipped if not installed? (#5224)

Reply:
Committed to now fail with: (not cleanest error but it's just tests)

>%Warning: Subprocess command `test.pl-file-needs-have_solver()-call' failed: